### DISCUSSION TITLE
rows range selection with shift key

### DISCUSSION THREAD

**Discussion by meddany** [KevinVandy/material-react-table#769] [Q&A] [answered]
Hello world,

is there is an option to be defined to apply rows range selection with shift key ?

**Comment by KevinVandy** [comment, marked as answer]
This is something that should be added to TanStack Table itself. We've wanted to add it for a while.

**Reply by meddany** [reply]
do we have a feedback from their side, if they are planning to add it ?

**Reply by KevinVandy** [reply]
Well I'm one of the maintainers for TanStack Table, so I'll see what we can do.
